src/DrupalPatchUtils/Command/CleanupGitBranches.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file
+ * Contains \DrupalPatchUtils\Command\CleanupGitBranches.
+ */
+
+namespace DrupalPatchUtils\Command;
+
+use DrupalPatchUtils\DoBrowser;
+use DrupalPatchUtils\DoFormBase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class CleanupGitBranches extends CommandBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure()
+  {
+    $this
+      ->setName('cleanupGitBranches')
+      ->setAliases(['cgb'])
+      ->setDescription('Removes all git branches which has closed issues.');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $process = new Process('git branch');
+    $process->run();
+
+    $branches = array_map('trim', explode("\n", $process->getOutput()));
+
+    $browser = new DoBrowser();
+    if (!$browser->loggedIn()) {
+      $browser->login($this->getConfig()->getDrupalUser(), $this->ask($output, "Enter your Drupal.org password: ", '', TRUE));
```

Comment:
This  needs to use the new tfa supporting code. Just do $browser = $this->login(OutputInterface $output);